gulpfile.js

```diff
@@ -11,8 +12,17 @@ theo.registerValueTransform(
   (prop) => prop.get('type') === 'color',
   (prop) => prop.getIn(['meta', 'filter']) || 'none',
 );
+
 theo.registerTransform('filter', ['filter']);
 
+theo.registerValueTransform(
+  'timing/ms-unitless',
+  (prop) => prop.get('type') === 'time',
+  (prop) => (prop.get('value') === 0 ? 0 : ms(prop.get('value'))),
```

Comment:
I just realized we also need a test that ensures that in `duration.raw.json`, `duration-none`'s value should be unitless, while the other values have `ms` in them.

Reply:
Incoming new commit! Please take a look and let me know if I did it correctly :)

Reply:
Looks good! 👍

Reply:
Looks good! 👍